Integration tests don't run if port is being used

Cypress tests should be able to run on another port when 8888 is used.
Trying to run `make test-ui-debug` command while lab is running would fail.

<details><summary>Command Line Output</summary>
<pre>
npm run test:integration:debug

```
> elyra@1.4.0-dev test:integration:debug /Users/kspuldaro/github/elyra
> start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:open
```
```
starting server using command "npm run start-jupyter"
and when url "http-get://localhost:8888?token=test" is responding
running tests using command "cy:open"
```
```
> elyra@1.4.0-dev start-jupyter /Users/kspuldaro/github/elyra
> mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py
```
```
[I 17:12:08.948 LabApp] The port 8888 is already in use.
[C 17:12:08.948 LabApp] ERROR: the notebook server could not be started because port 8888 is not available.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! elyra@1.4.0-dev start-jupyter: `mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the elyra@1.4.0-dev start-jupyter script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kspuldaro/.npm/_logs/2020-10-27T21_12_09_022Z-debug.log
Error: server closed unexpectedly
    at ChildProcess.onClose (/Users/kspuldaro/github/elyra/node_modules/start-server-and-test/src/index.js:60:14)
    at ChildProcess.emit (events.js:321:20)
    at maybeClose (internal/child_process.js:1026:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! elyra@1.4.0-dev test:integration:debug: `start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:open`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the elyra@1.4.0-dev test:integration:debug script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kspuldaro/.npm/_logs/2020-10-27T21_12_09_097Z-debug.log
make: *** [test-ui-debug] Error 1
```
</pre>

--- a/cypress.json
+++ b/cypress.json
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://localhost:8888/lab",
+  "baseUrl": "http://localhost:58888/lab",
   "integrationFolder": "./tests/integration",
   "supportFile": false,
   "pluginsFile": "./tests/plugins/index.js",

--- a/package.json
+++ b/package.json
@@ -15,8 +15,8 @@
     "prettier": "prettier --ignore-path .gitignore --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "start-jupyter": "mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py",
-    "test:integration": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:run",
-    "test:integration:debug": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:open",
+    "test:integration": "start-server-and-test start-jupyter http-get://localhost:58888?token=test cy:run",
+    "test:integration:debug": "start-server-and-test start-jupyter http-get://localhost:58888?token=test cy:open",
     "test:unit": "lerna run test --scope \"@elyra/*\" --concurrency 1 --stream",
     "test": "npm run test:unit && npm run test:integration"
   },

--- a/tests/test-config.py
+++ b/tests/test-config.py
@@ -16,6 +16,7 @@
 c.Session.debug = True
 c.LabApp.token = 'test'
 c.LabApp.open_browser = False
+c.NotebookApp.port = 58888
 c.NotebookApp.port_retries = 0
 c.LabApp.workspaces_dir = './build/cypress-tests'
 c.FileContentsManager.root_dir = './build/cypress-tests'

